app/_locales/de/messages.json

```diff
@@ -3186,20 +3186,12 @@
   },
   "permission_manageBip32Keys": {
     "message": "Verwalten Sie Ihre Konten und Assets unter $1 ($2).",
-    "description": "The description for the `snap_getBip32Entropy` permission. $1 is a derivation path, e.g. 'm/44'/0'/0''. $2 is the elliptic curve name, e.g. 'secp256k1'."
-  },
-  "permission_manageBip32KeysDescription": {
-    "message": "Gestatten Sie dem Snap, BIP-32-Schlüssel basierend auf Ihrer geheimen Wiederherstellungsphrase abzuleiten, ohne diese offenzulegen. Dies gewährt vollständigen Zugriff auf alle Konten und Assets auf $1.\nMit der Berechtigung zur Verwaltung von Schlüsseln kann das Snap eine Vielzahl von Blockchain-Protokollen über Ethereum (EVMs) hinaus unterstützen.",
-    "description": "An extended description for the `snap_getBip32Entropy` permission. $1 is a derivation path (name)"
+    "description": "The description for the `snap_getBip32Entropy` permission. $1 is a derivation path, e.g. 'm/44'/0'/0' (secp256k1)'."
```

Comment:
All the descriptions for this key seems to be outdated? It still has `$2` that is the curve. Could we revert?

Reply:
This is automatically updated with `yarn verify-locales:fix`. `$2` is not used anymore for this because of how the strings are constructed now for Unknown networks, etc. (in a more nested way).
These translations are now incorrect. I'm not sure what to do with them?
It shouldn't be using `$2` and it should be just like this instead:
```json
  "permission_manageBip32Keys": {
    "message": "Manage $1 accounts.",
    "description": "The description for the `snap_getBip32Entropy` permission. $1 is a derivation path, e.g. 'm/44'/0'/0' (secp256k1)'."
  },
```
Because:
```javascript
return {
        ...baseDescription,
        label: t('permission_manageBip32Keys', [
          <Text
            as="span"
            fontWeight={FontWeight.Medium}
            className="permission-label-item"
            key={path.join('/')}
          >
            {`${t('unknownNetworkForKeyEntropy')} ${path.join('/')} (${curve})`}
          </Text>,
        ]),
        description: t('permission_manageBip44AndBip32KeysDescription'),
      };
```

Reply:
@david0xd If the translations are outdated, lets remove them and they can get retranslated.

Reply:
Done. All affected are removed now.